ktor-http/wasmJs/src/io/ktor/http/URLBuilderWasm.kt

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+*/
+
+package io.ktor.http
+
+import io.ktor.util.*
+
+private fun locationOrigin(): String = js(
```

Comment:
It looks like we have 2 copies of the same code, so it will be needed to copy bug fixes multiple times as well. Could we make an umbrella source set to share code between js and wasm and expect actual for the diference?

Reply:
Fixed